snap/snapcraft.yaml

```diff
@@ -1,52 +1,52 @@
-name: spark-k8s # you probably want to 'snapcraft register <name>'
-base: core20
-version: '0.1'
-summary: Client side snap to submit spark jobs to kubernetes.
-description: |
-  Spark is now available on Kubernetes and needs a client to submit and 
-  interact with jobs. This snap packages the client binaries for the same.
-
-grade: devel
-confinement: strict
-
-apps:
-  spark-submit:
-    command: bin/spark-submit
-    environment:
-      JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64
-      PATH: $JAVA_HOME/bin:$PATH
-  spark-shell:
-    command: bin/spark-shell
-    environment:
-      JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64
-      PATH: $JAVA_HOME/bin:$PATH
-  pyspark:
-    command: bin/pyspark
-    environment:
-      PYTHONPATH: $PYTHONPATH:$SNAP/usr/lib/python3/dist-packages
-
-parts:
-  spark:
-    source: https://github.com/apache/spark.git
-    plugin: nil
-    build-packages: 
-        - git
-        - openjdk-11-jdk
-        - maven
-        - python3
-        - python3-pip
-        - wget
-    stage-packages:
-        - openjdk-11-jre
-    override-build: |
-        dev/make-distribution.sh -Pkubernetes
-        cd dist/jars
-        wget https://repo1.maven.org/maven2/net/java/dev/jets3t/jets3t/0.9.4/jets3t-0.9.4.jar
-        wget https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.11.874/aws-java-sdk-bundle-1.11.874.jar
-        wget https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.2.2/hadoop-aws-3.2.2.jar
-        cd ../..
-        mkdir -p $SNAPCRAFT_PRIME/bin
-        cp -r dist/bin/* $SNAPCRAFT_PRIME/bin/
-        mkdir -p $SNAPCRAFT_PRIME/jars
-        cp -r dist/jars/* $SNAPCRAFT_PRIME/jars/
+name: spark-k8s 
+base: core20 
+version: '0.1' 
+summary: Client side snap to submit spark jobs to kubernetes. 
+description: | 
+  Spark is now available on Kubernetes and needs a client to submit and  
+  interact with jobs. This snap packages the client binaries for the same. 
+ 
+grade: devel 
+confinement: strict 
+ 
+apps: 
+  spark-submit: 
+    command: bin/spark-submit 
+    environment: 
+      JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64 
+      PATH: $JAVA_HOME/bin:$PATH 
+  spark-shell: 
+    command: bin/spark-shell 
+    environment: 
+      JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64 
+      PATH: $JAVA_HOME/bin:$PATH 
+  pyspark: 
+    command: bin/pyspark 
+    environment: 
+      PYTHONPATH: $PYTHONPATH:$SNAP/usr/lib/python3/dist-packages 
+ 
+parts: 
+  spark: 
+    source: https://github.com/apache/spark.git 
```

Comment:
Use a dedicated mirror of spark with the pom.xml adapted to integrate the dependencies, rather than using wget